README.md

```diff
@@ -31,6 +31,45 @@ prass convert-srt input.srt | prass copy-styles --from template.ass | prass sort
 ```
 If you don't provide some file arguments, most commands will use stdin/stdout by default. They will also do this if you provide "-" as file paths.
 
+### Python scripting
```

Comment:
Prass is an application, not a library. You can use it as such as any other python script, but I will never support or endorse it in any way.